### ISSUE TITLE
Baxxid unable to drop things

### ISSUE THREAD

**Issue by TheKnightofAura** [ScavStation/Baystation12#223]
#### Description of issue
Baxxid, lacking manipulation, cannot remove backpacks, or any other item with an inventory.

#### Difference between expected and actual behavior
baxxid should be able to take off anything they can put on themselves, as well as put things into inventories.

#### Steps to reproduce

Be a Baxxid with a backpack.
Try for several minutes to be rid of it by yourself.

#### Specific information for locating

Backpack, Baxxid

#### Length of time in which bug has been known to occur

Since baxxid, I guess?

#### Client version, Server revision & Game ID

Meh

#### Issue bingo

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/Baystation12/Baystation12/issues)

**Comment by SodiumShine** [comment]
Can take my backpack off just fine as a baxxid after testing multiple times. Just drag and drop bag off its slot

**Comment by TheKnightofAura** [comment]
Yup The issue for removing was fixed 10 days ago, pretty much, in the merge. Not fully sure why it's still up? Maybe for other reasons?

**Comment by SodiumShine** [comment]
@MistakeNot4892 please confirm

**Comment by MistakeNot4892** [comment]
That's a workaround (and due to a bug), this still needs a proper fix.

**Comment by MistakeNot4892** [comment]
Appears to be properly broken now, can't take the backpack off even with the workaround.

**Comment by Qumefox** [comment]
I just confirmed that this is still a valid issue. However i'm not sure what solution to offer, given the.. uniqueness.. of not having hands.
